concepticondata/conceptlists/Verheyen-2019-1000.tsv

```diff
@@ -0,0 +1,1001 @@
+Verheyen-2019-1000-281	281	gelukkig	lucky			5.3	5.85	4.5	5.65	5.9	5.35	6.35	8	3	48506	0	7396	10396	5.59557409789311	0.000647800348354
+Verheyen-2019-1000-282	282	gemakkelijk	easy	EASY	686	4.95	3.3	3.2	3.8	6.35	4.1	5.15	11	4	83275	0	7535	2085	5.79896785491736	0.001009615545239
+Verheyen-2019-1000-283	283	gemeen	nasty			5.8	4.5	3.8	4.6	5.1	4.85	2	6	2	70993	1	1691	2732	5.25256362713423	0.000821150650296
+Verheyen-2019-1000-284	284	gemeenschappelijk	common			9.2	4.4	5.45	3.5	4.7	3.45	4.75	17	5	162739	0	1997	223	0.310359347214923	5.60044808791395E-05
+Verheyen-2019-1000-285	285	gemengd	mixed			7.05	3.95	3.7	3.75	4.15	5.45	4.75	7	2	75586	0	463	22	0.194079527198336	0.000135217791371
+Verheyen-2019-1000-286	286	gemiddeld	average			9.2	3.7	4.85	3.7	4.35	3.75	4.1	9	3	66108	1	900	427	0.788410318907814	0.000127163689675
+Verheyen-2019-1000-287	287	geneeskundig	medical			11	4.05	5.2	4.4	3.45	5.25	4.35	12	4	100411	0	158	18	0.044870997133944	5.5256018903416E-06
+Verheyen-2019-1000-288	288	geniaal	briljant			9.75	5.45	3.5	6.3	4.8	3.8	5.25	7	3	77515	0	417	730	0.316432760559233	4.85003360268853E-05
+Verheyen-2019-1000-289	289	genoeg	enough	ENOUGH	1881	4.7	3.6	2.9	3.89473684210526	5.75	3.1	4.15	6	2	64482	1	0	76	2.48242129118543	0.000332964699539
+Verheyen-2019-1000-290	290	genotypisch	genotypical			15.3888888888889	4	4.94117647058824	3.88888888888889	1.85714285714286	3.28571428571429	4.18181818181818	11	4	90064	0	7	0	NaN	NaN
+Verheyen-2019-1000-291	291	gerespecteerd	respected			9.6	4.05	4.6	5.25	4.25	4.75	5.85	13	4	167692	0	0	0	NaN	NaN
+Verheyen-2019-1000-292	292	gering	little	A LITTLE	2924	10.95	3.22222222222222	3.61111111111111	2.89473684210526	3.3	3.15789473684211	3.10526315789474	6	2	101142	8	3539	120	0.198201637594148	4.4478166452997E-05
+Verheyen-2019-1000-293	293	geringer	smaller			11.65	2.86666666666667	3.6	3	2.36842105263158	3.76470588235294	3.25	8	3	155835	0	3539	0	0.004484304932735	2.8766973327809E-06
+Verheyen-2019-1000-294	294	geringst	smallest			12.2105263157895	3.42857142857143	3	2.76470588235294	2.45	3.26315789473684	3.15	8	2	123775	0	3539	0	NaN	NaN
+Verheyen-2019-1000-295	295	gesloten	closed	CLOSED	589	5.8	2.45	4.5	3.5	5.6	6.4	2.9	8	3	86995	1	1927	84	2.35319740493442	0.000381888079292
+Verheyen-2019-1000-296	296	gestorven	dead	DEAD	1625	5.75	1.9	6.05	3.35	4.47368421052632	6.7	1.3	9	3	102103	1	0	0	0.202329364084863	1.83015223976921E-05
+Verheyen-2019-1000-297	297	geurig	fragrant	FRAGRANT	376	8.8421052631579	4.9	4.4	4.05263157894737	3.85	4.75	5.2	6	2	41385	2	371	29	0.727697119310004	5.59459055940829E-05
+Verheyen-2019-1000-298	298	gevoelig	sensitive			7.15	4.4	3.85	4.05	5.65	5.6	4.95	8	3	68174	0	1547	1278	2.8421289780418	0.000419225484605
+Verheyen-2019-1000-299	299	gevorderd	advanced			9.7	5.15	5	5.45	3	3.75	5.3	9	3	94079	0	201	2	0.004761904761905	9.69746385937454E-07
+Verheyen-2019-1000-300	300	gevuld	full	FULL	1429	6.2	3.8	4.6	4.2	4.05	5.9	4.68421052631579	6	2	22221	1	684	13	0.264969196380376	9.22267863216556E-05
```

Comment:
gevuld is filled or stuffed; again, not sure about whether the semantic difference to the concept is acceptable or not